tests/jax/systems/ippo_system_test.py

```diff
@@ -106,14 +105,8 @@ def network_factory(*args: Any, **kwargs: Any) -> Any:
     trainer_node.disable_run()
```

Comment:
As discussed, lets disable all nodes and call them manually so we have control over execution (for all integration tests)